bindings/node/test/clientEncryption.test.js

```diff
@@ -186,3 +189,143 @@ describe('ClientEncryption', function() {
 
   it('should be able to auto decrypt explicit encryption');
 });
+
+describe('ClientEncryptionKeyAltNames', function() {
```

Comment:
should this just be a `describe` block within `ClientEncryption` called `keyAltNames`?

Reply:
I specifically removed it b/c the `before` block inside `ClientEncryption` is stubbing out `stateMachine.prototype.kmsRequest`. The other two options would be:

a. move all of the other tests into another describe block that includes stubbing `stateMachine.prototype.kmsRequest` so that it only applies to the tests that use it
b. clearing the stub once we get to the `keyAltNames` tests.

I'm okay with any of the above approaches.

Reply:
I'm okay with either as well